src/boot/bootctl.c

```diff
@@ -589,10 +589,9 @@ static int print_efi_option(uint16_t id, int *n_printed, bool in_order) {
                 return log_debug_errno(r, "Failed to read boot option 0x%04X: %m", id);
 
         /* print only configured entries with partition information */
-        if (!path || sd_id128_is_null(partition)) {
-                log_debug("Ignoring boot entry 0x%04X without partition information.", id);
-                return 0;
```

Comment:
a future use might not do the same, so better to keep returning 0 here to avoid future issues

Reply:
This is not a library function, but only used in a specific place of bootctl. Hence it is not necessary to consider any future usages.

But, OK, this change is not important for this PR. I will drop it.